packages/node_modules/@webex/internal-plugin-user/src/user.js

```diff
@@ -290,8 +290,7 @@ const User = WebexPlugin.extend({
     }
 
     return this.webex.internal.services.collectPreauthCatalog({email})
-      .then(() => this.webex.credentials.getUserToken())
```

Comment:
Was this change supposed to be in this PR?

Reply:
no... must have been a bad merge or something on my part... my mistake. removing.